src/main/java/nl/miraclethings/tools/ui/LayoutUtil.java

```diff
@@ -22,4 +25,11 @@ public static int getActionBarHeight(Context context) {
         return mActionBarSize;
 
     }
+    
+    public static void setTextOrHide(TextView textView, String content){
+        if(textView != null){
+            textView.setText(content);
+            textView.setVisibility(StringUtils.isEmpty(content)?View.GONE: View.VISIBLE);
```

Comment:
Commons lang is not standard on Android, IIRC. If you use [TextUtils.isEmpty](https://developer.android.com/reference/android/text/TextUtils.html#isEmpty(java.lang.CharSequence)), it does not require an external library.